pom.xml

```diff
@@ -589,6 +598,69 @@
       </build>
     </profile>
 
+    <profile>
+      <id>clirr-compatibility-check</id>
```

Comment:
Would it make sense to combine `clirr-compatibility-check` and `animal-sniffer` under one profile like `compatibility-checks`?